Gemfile

```diff
@@ -2,7 +2,8 @@ source 'https://rubygems.org'
 
 gem 'rspec'
 gem "rack"
-gem "sinatra"
+gem "sinatra", "~> 2.0.0.rc2"
```

Comment:
Maybe something like [Appraisal](https://github.com/thoughtbot/appraisal) could be used